03-quickstart.rst

```diff
@@ -200,29 +200,34 @@ package. Here is a minimal setup that should open a window with a black backgrou
 
 .. code:: python
 
-   import sys
    import OpenGL.GL as gl
    import OpenGL.GLUT as glut
 
+   window = None
+
+
    def display():
-       gl.glClear(gl.GL_COLOR_BUFFER_BIT)
```

Comment:
Why did you remove the clear?

Reply:
I am not sure now. I will add it back in an update the PR.